client/src/components/relatedProducts/Modal.jsx

```diff
@@ -9,38 +10,69 @@ const Modal = (props) => {
   if (comparedProduct.product !== undefined && comparedProduct.product.features !== undefined) {
     // eslint-disable-next-line no-plusplus
     for (let i = 0; i < currentProduct.features.length; i++) {
-      if (currentProduct.features[i].value === true) {
-        currentProduct.features[i].value = <FontAwesomeIcon icon={'faCheck'} />;
-      } else if (currentProduct.features[i].value === false) {
-        currentProduct.features[i].value = '';
-      }
       features[currentProduct.features[i].feature] = {
         currentProduct: currentProduct.features[i].value,
       };
     }
     // eslint-disable-next-line no-plusplus
     for (let j = 0; j < comparedProduct.product.features.length; j++) {
-      if (comparedProduct.product.features[j].value === true) {
-        comparedProduct.product.features[j].value = <FontAwesomeIcon icon={'faCheck'} />;
-      } else if (comparedProduct.product.features[j].value === false) {
-        comparedProduct.product.features[j].value = '';
+      if (!features[comparedProduct.product.features[j].feature]) {
+        features[comparedProduct.product.features[j].feature] = {
+          comparedProduct: comparedProduct.product.features[j].value,
+        };
+      } else {
+        features[comparedProduct.product.features[j].feature].comparedProduct = (
+          comparedProduct.product.features[j].value
+        );
       }
-      features[comparedProduct.product.features[j].feature] = {
-        comparedProduct: comparedProduct.product.features[j].value,
-      };
     }
   }
 
   const keys = Object.keys(features);
   // eslint-disable-next-line react/jsx-key
-  const rows = keys.map((key) => (
-      // eslint-disable-next-line react/jsx-key
-      <tr className='features'>
-        <td className='check'>{features[key].currentProduct}</td>
-        <td>{key}</td>
-        <td className='check'>{features[key].comparedProduct}</td>
+  const rows = keys.map((key) => {
+    if (features[key].currentProduct === true && features[key].comparedProduct === true) {
```

Comment:
Just my preference though:

```suggestion
    if (features[key].currentProduct && features[key].comparedProduct) {
```

Reply:
This was intentional because I don't want a truthy. I only want the check to render if the value is a boolean, specifically true.